example/ionic-angular/README.md

```diff
@@ -33,3 +33,17 @@ Then on the ionic-angular folder
 ### Web
 
 `http-server ./www`
+
+# Troubleshooting
```

Comment:
This is for ppl using the sdk or developing? If it's for users it should be on the docs Instead

Reply:
It could happen with anyone trying a capacitor project on a Mac M1.

It's not a problem with the SDK but more like a guide to folks trying the examples on an ARM device.

Reply:
I think that should be pointed out. It can be confusing: Troubleshooting if you're using the SDK?